proactor/README.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: pattern title: Proactor folder: proactor permalink: /patterns/proactor/ categories: Concurrency tags:
+
+- Reactive
+
+---
```

Comment:
The line endings are not functioning.

Reply:
Add `language: en` as instructed in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

Reply:
add and change